CONTRIBUTING.md

```diff
@@ -1,21 +1,129 @@
-# Contributing
+# How to contribute
 
-## Evolving the Format of the Forecast Data
+Soaringmeteo team is glad you are reading this, because we always need volunteers
+to keep this awesome service up to date.
 
-The frontend consumes the forecast data produced by the backend, so both parties need to agree on their format.
-Furthermore, the frontend also reads several forecast data in the past (users can show the data from the latest
-forecast run, or from older runs as well). This has consequences on how the format of the forecast data produced
-by the backend can evolve:
+* Find a bug? Do not hesitate to open an issue using the bug template. **TODO: LINK HERE**
+* Have an awesome improvement idea? Do not hesitate to open an issue using the feature template **TODO: LINK HERE**
```

Comment:
I think the link should be: https://github.com/soaringmeteo/soaringmeteo/issues/new/choose